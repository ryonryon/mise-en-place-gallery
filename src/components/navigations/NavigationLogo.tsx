import styled from "@emotion/styled";
import * as React from "react";

interface Props {
  logo: string;
}

const LogoDiv = styled.div`
  padding: 24px 0 24px 16px;
  font-size: 24px;
`;

const NavigationLogo = ({ logo }: Props) => (
  <a href="">
    <LogoDiv>{logo}</LogoDiv>
  </a>
);

export default NavigationLogo;
