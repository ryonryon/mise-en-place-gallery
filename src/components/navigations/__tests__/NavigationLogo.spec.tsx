import * as React from "react";
import * as renderer from "react-test-renderer";

import NavigationLogo from "../NavigationLogo";

describe("NavigationLogo test", () => {
  it("tests NavigationLogo render correctly", () => {
    const navigationLogo = renderer.create(<NavigationLogo logo="test" />);

    expect(navigationLogo).toMatchSnapshot();
  });
});
