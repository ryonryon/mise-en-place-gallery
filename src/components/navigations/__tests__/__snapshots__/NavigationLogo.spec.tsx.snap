// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationLogo test tests NavigationLogo render correctly 1`] = `
<a
  href=""
>
  <div
    className="css-3ihe6y"
  >
    test
  </div>
</a>
`;
